dashboard/db/migrate/20160504000000_migrate_net_promoter_score2015_data.rb

```diff
@@ -0,0 +1,39 @@
+# A migration that transfers the net promoter score survey results in the users
+# table to the survey_results table. Does NOT remove the data from the users
+# table.
+class MigrateNetPromoterScore2015Data < ActiveRecord::Migration
+  def change
+    reversible do |direction|
+      # In the up direction, we create SurveyResult records for users having
+      # a survey2015_value or survey2015_comment. For safety, we do not remove
+      # the information from the users row in this migration.
+      direction.up do
+        ActiveRecord::Base.record_timestamps = false
+        begin
+          User.all.each do |user|
```

Comment:
User.all is not a good idea because it will try to create an ActiveRecord object for every user (all at once..). Try find_each (http://apidock.com/rails/ActiveRecord/Batches/find_each) to do this in batches.